src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityLink.tsx

```diff
@@ -17,28 +17,33 @@ const AnnotationActivityLink = ({
     message,
     onClick = noop,
 }: AnnotationActivityLinkProps): JSX.Element => {
+    const handleClick = (event: React.SyntheticEvent<HTMLButtonElement>) => {
```

Comment:
Should this also have a `if (isDisabled) { return; }` guard clause?

Reply:
The behavior for clicks when disabled is handled by the PlainButton component. This is pretty much how the logic was before.